lib/charms/ovn_charm.py

```diff
@@ -311,6 +311,14 @@ def configure_tls(self, certificates_interface=None):
         # The default handler in ``OpenStackCharm`` class does the CA only
         tls_objects = self.get_certs_and_keys(
             certificates_interface=certificates_interface)
+        if not tls_objects:
+            # We have no configuration settings ssl_* nor a certificates
```

Comment:
FWIW, the ssl_* config options don't exist on the OVN charms b/c they don't provide a public interface and they need a unique cert per unit AIUI, therefore Vault is used for this scenario since it can provide the cert per unit where a config option is awkward.